riak/src/main/java/com/yahoo/ycsb/db/RiakKVClient.java

```diff
@@ -0,0 +1,493 @@
+/*
+ * Copyright 2016 nygard_89
```

Comment:
We try to use a blanket of "YCSB contributors" for the changes that come directly to the project. Generally we then rely on git to provide us with who made a given line.

I agree that we need to leave the Basho line in, similar to how we handle the original Yahoo! Lines. Would you mind changing the bit with your GitHub handle to instead be the contributors reference?